docs: improve network usage instructions (#941)
succinctlabs/sp1: book/prover-network/setup.md

## Get access

> **[FILL OUT THIS FORM](https://forms.gle/rTUvhstS8PFfv9B3A)** to gain access to the Succinct

Reviewer: IMO this was better as it was before

Author: @ratankaliani said that it was too hard to find the form

Network. Completing this form requires you to complete the [key setup](#key-setup) steps below.

### Key Setup

The prover network uses secp256k1 keypairs for authentication, like Ethereum wallets. You may generate a new keypair explicitly for use with the prover network, or use an existing keypair. Currently you do not need to hold any funds in this account, it is used solely for access control.

Prover network keypair credentials can be generated using the
[cast](https://book.getfoundry.sh/cast/) CLI tool.

First [install Foundry](https://book.getfoundry.sh/getting-started/installation#using-foundryup):

```sh
curl -L https://foundry.paradigm.xyz | bash
```

> after running this command, you will be prompted to source your shell profile and run `foundryup`.
> After that completes, you will have access to the `cast`.

Reviewer: ```suggestion
Upon running this command, you will be prompted to source your shell profile and run `foundryup`. Afterwards you should have access to the `cast` command.
```
